packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -31,6 +31,9 @@ html.interface-interface-skeleton__html-container {
 			top: 0;
 		}
 	}
+
+	// This default value avoids widows in multi-line paragraphs of text.
+	text-wrap: pretty;
```

Comment:
If we want this to be really global, maybe we should add it to the "reset" mixin?

Reply:
Good idea. Pending other thoughts, I'll move it there.